File: javascript/src/models/status.ts
// this file is @generated

export enum Status {
  Pending = "pending",
  Active = "active",
}

export const StatusSerializer = {
  _fromJsonObject(object: any): Status {
    return object;
  },

  _toJsonObject(self: Status): any {
    return self;
  },
};

File: rust/src/models/status.rs
// this file is @generated
use std::fmt;

use serde::{Deserialize, Serialize};

#[derive(Clone, Copy, Debug, Eq, PartialEq, Ord, PartialOrd, Hash, Serialize, Deserialize)]
pub enum Status {
    #[serde(rename = "pending")]
    Pending,
    #[serde(rename = "active")]
    Active,
}

impl fmt::Display for Status {
    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
        let value = match self {
            Self::Pending => "pending",
            Self::Active => "active",
        };
        f.write_str(value)
    }
}
